website/docs/Examples/Integrate - Scikit-learn Pipeline.md

```diff
@@ -1,6 +1,11 @@
 As FLAML's AutoML module can be used a transformer in the Sklearn's pipeline we can get all the benefits of pipeline.
 
-This example requires openml==0.10.2.
+### Prerequisites
+
+Install the [automl] option.
+```bash
+pip install "flaml[automl] openml==0.10.2"
```

Comment:
We do need to update the version of openml as in conda forge, the oldest version is 0.11.0

https://anaconda.org/conda-forge/openml/files